src/lib/openapi/spec/project-overview-schema.ts

```diff
@@ -17,42 +17,60 @@ export const projectOverviewSchema = {
     properties: {
         stats: {
             $ref: '#/components/schemas/projectStatsSchema',
+            description: 'Project statistics',
         },
         version: {
             type: 'number',
+            example: 1,
+            description: '',
         },
         name: {
             type: 'string',
+            example: 'dx-squad',
+            description: 'Name of this project used to identify it',
         },
         description: {
             type: 'string',
             nullable: true,
+            example: 'DX squad feature release',
+            description: 'Additional information about the project',
         },
         members: {
             type: 'number',
+            example: 4,
+            description: 'Count of the project members',
         },
         health: {
             type: 'number',
+            example: 50,
+            description: 'Indicator of the project health in the 0-100 scale',
```

Comment:
How about adding a doc link here?

```suggestion
            description: 'An indicator of the [project's health](https://docs.getunleash.io/reference/technical-debt#health-rating) on a scale from 0 to 100'
```